sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py

```diff
@@ -428,13 +428,13 @@ def __init__(self, method_name):
             service_endpoint = self.get_settings_value("METRICS_ADVISOR_ENDPOINT")
             subscription_key = self.get_settings_value("METRICS_ADVISOR_SUBSCRIPTION_KEY")
             api_key = self.get_settings_value("METRICS_ADVISOR_API_KEY")
-            self.anomaly_detection_configuration_id = self.get_settings_value("ANOMALY_DETECTION_CONFIGURATION_ID")
-            self.anomaly_alert_configuration_id = self.get_settings_value("ANOMALY_ALERT_CONFIGURATION_ID")
-            self.metric_id = self.get_settings_value("METRIC_ID")
-            self.incident_id = self.get_settings_value("INCIDENT_ID")
-            self.dimension_name = self.get_settings_value("DIMENSION_NAME")
-            self.feedback_id = self.get_settings_value("FEEDBACK_ID")
-            self.alert_id = self.get_settings_value("ALERT_ID")
+            self.anomaly_detection_configuration_id = "c0f2539f-b804-4ab9-a70f-0da0c89c76d8"
+            self.anomaly_alert_configuration_id = "ff3014a0-bbbb-41ec-a637-677e77b81299"
+            self.metric_id = "3d48ed3e-6e6e-4391-b78f-b00dfee1e6f5"
```

Comment:
Why we hard code the id here?

Reply:
So we don't have to add them to the test secrets keyvault.

Reply:
But if we modify the secrets that we saved in keyvault, we have to update them as well, right?

I mean these ids are coupled with other secrets.

Reply:
yeah that's a good point. We were trying to minimize the number of things we were adding to the keyvault, but I think it probably makes more sense to allow ease of update here. I'll add them back as env vars.